packages/gleam/build.sh

```diff
@@ -2,11 +2,12 @@ TERMUX_PKG_HOMEPAGE=https://gleam.run
 TERMUX_PKG_DESCRIPTION="A friendly language for building type-safe, scalable systems!"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.6.1"
+TERMUX_PKG_VERSION="1.6.2"
 TERMUX_PKG_SRCURL=https://github.com/gleam-lang/gleam/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=86bdaa4b7803230bd434044e0632f6d31246a8514fd68d10c3add9dfff400544
+TERMUX_PKG_SHA256=12db37408b69ef224d7d77fa6e7422124f6e8956419c48202de41ce182fb075d
 TERMUX_PKG_DEPENDS="erlang"
-TERMUX_PKG_SUGGESTS="nodejs"
+TERMUX_PKG_SUGGESTS="nodejs | nodejs-lts"
+TERMUX_PKG_ANTI_BUILD_DEPENDS="nodejs, nodejs-lts"
```

Comment:
With these it's fine to keep open since there's packaging changes besides the update.